eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -206,7 +207,8 @@ protected function indexIncrementally(InputInterface $input, OutputInterface $ou
         }
 
         $iterations = ceil($count / $iterationCount);
-        $processCount = (int) $input->getOption('processes');
+        $processes = $input->getOption('processes');
+        $processCount = $processes === 'auto' ? $this->getNumberOfCPUCores() - 1 : (int) $processes;
         $processCount = $processCount > $iterations ? $iterations : $processCount;
```

Comment:
Tip for future: You can replace `$processCount > $iterations ? $iterations : $processCount` with `min($processCount, $iterations)`

Reply:
thanks, fixed